src/System.Net.Security/tests/UnitTests/System.Net.Security.Unit.Tests.csproj

```diff
@@ -9,6 +9,8 @@
       imported. We are disabling the "Type conflicts with the imported type CS0436 warning" for these types of projects.
     -->
     <NoWarn>436</NoWarn>
+    <!-- Disable: CLSCompliant attribute is not needed -->
+    <NoWarn>$(NoWarn);3021</NoWarn>
```

Comment:
What's causing the warning to show up here but not in the src project?

Reply:
I haven't looked into that. Perhaps this is already ignored there already

Reply:
Let's figure it out before checking this in.

Reply:
Ok, https://github.com/dotnet/corefx/blob/master/Directory.Build.props#L321 - we mark product as ClsCompliant but not tests. Warning is complaining that:
`error CS3021: 'SslStream.NegotiatedCipherSuite' does not need a CLSCompliant attribute because the assembly does not have a CLSCompliant attribute`

in this proj we include SslStream.cs which contains CLSCompliantAttribute and since proj is not marked as such it complains

Reply:
Ok. That's fine then. Thank you for checking.